RELEASES.md

```diff
@@ -1,3 +1,353 @@
+Version 1.7.0 (2016-03-03)
```

Comment:
This is a good idea, but I think I won't do it. It would require patching both the beta and master branches later.